src/components/Table/GridTable.stories.tsx

```diff
@@ -950,3 +951,80 @@ export function StickyColumnsAndHeader() {
     </div>
   );
 }
+
+export function ActiveRow() {
+  const api = useRef<GridTableApi<Row> | undefined>();
```

Comment:
You could remove this `useRef` and use the `data` params?

Reply:
Oh wait, this is a row click, gotcha...

Reply:
Yeah, I could. I did it that way for the NestedActiveRow story and wanted to see how it'd work with the `rowStyles.onClick` -> Perhaps we should pass the `api` back in that function too?